ffmpeg.py

```diff
@@ -19,7 +34,9 @@ def get_packets_info(file_path):
     args = shlex.split(cmd)
     args.append(file_path)
     output = subprocess.check_output(args, stderr=subprocess.DEVNULL)
-    output = json.loads(output)
+    output = json.loads(output)[packets_str]
```

Comment:
So you can actually do all the show_x in one invocation.
-show_packets -show_format -show_streams.

It doesnt matter too much to invoke it multiple times though.

Reply:
Ok. I will try to refactor the code in latter cl to cut off redundant invocation.